app/controllers/services_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ServicesController < ApplicationController
   def show
     @service = Service.find(params[:id])
+    @interactions = Interaction.joins(:service_interactions).where('service_interactions.service_id': @service.id)
```

Comment:
It feels like this is the sort of association walking we'll want to do a bunch, so it'll be useful to add these as activerecord associations to the `Service` and `Interaction` models to make it easier.

```ruby
class Service
  has_many :service_interactions
  has_many :interactions, through: :service_interactions
end
```

And the same for `Interaction` to get all the `Service`s that implement it.